redfish-core/include/utils/hw_isolation.hpp

```diff
@@ -57,29 +125,33 @@ inline void
             BMCWEB_LOG_ERROR << "DBus ErrorName: " << dbusError->name
                              << " ErrorMsg: " << dbusError->message;
 
+            // The Enabled property value will be "false" to isolate.
+            constexpr bool enabledPropVal = false;
             if (strcmp(dbusError->name, "xyz.openbmc_project.Common.Error."
                                         "InvalidArgument") == 0)
             {
-                constexpr bool isolate = false;
-                messages::propertyValueIncorrect(aResp->res, "@odata.id",
-                                                 std::to_string(isolate));
+                messages::propertyValueIncorrect(
+                    aResp->res, "@odata.id", std::to_string(enabledPropVal));
```

Comment:
The backend will expect the resource inventory path to isolate and that's might be wrong based on the backend end validation and return an `InvalidArgument` error.

Here, the bmcweb returning `The property 'Enabled' with the requested value of 'false' could not be written because the value does not meet the constraints of the implementation` means the user might get confused, right? I will agree, In general, if the user assumed that passed property value is wrong if we returned `propertyValueIncorrect` error. 

So, do we need to use `ResourceNotFound` in this case since the backend D-Bus API expects inventory path alone for the manual operation (since nothing is wrong in the "Enabled" property value because it is like toggle (true/false)? 

In the future, the backed expecting more arguments then, we might need to adjust that here too i.e communicate to the user like which argument is wrong.